templates/partials/global/error.html

```diff
@@ -0,0 +1,7 @@
+<h1>{{error.output.payload.status}} {{error.output.payload.error}}</h1>
+{{#if error.output.payload.message}}
+  <p>{{error.output.payload.message}}</p>
+{{/if}}
+{{#if isDevelopment}}
+  <pre>{{error.stack}}</pre>
+{{/if}}
```

Comment:
No need to pull this out into a partial right now - only the error page needs to use it